strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/MutableStorage.java

```diff
@@ -87,7 +88,13 @@ public void setBasedir(String basedir)
 
     public void initDefaultBasedir(String id)
     {
-        Path basedirPath = Paths.get(System.getProperty("strongbox.storage.booter.basedir"));
+        //TODO: we should rework this to use SpringBoot environment instead of `System.getProperty` ASAP!!!
```

Comment:
@carlspring the follow up task should be to rework this method.

Reply:
@ptirador ,

Would you like to look into this?

Reply:
@sbespalov Do you mean to inject `Environment` class from `Environment API`, and then use its `getProperty` method?

@Inject

Reply:
@ptirador the thing is that we can't inject anything here, because this class is not under Spring context. So we should drop this `MutableStorage.initDefaultBasedir()` method and find appropriate solution how to set `MutableStorage.basedir` property from Spring environment.